velox/sdk/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+find_library(RE2 re2 REQUIRED)
+#set(ENV{JAVA_HOME} "/Users/xuyiming/Library/Java/JavaVirtualMachines/openjdk-18.0.2/Contents/Home")
+#set(ENV{JAVA_HOME} "/Library/Java/JavaVirtualMachines/jdk1.8.0_361.jdk/Contents/Home")
```

Comment:
Things like this should be provided by the user and never hardcoded. Giving an error or warning if not set during configure is fine though.